### DISCUSSION TITLE
Manually specify rest time based on exercise

### DISCUSSION THREAD

**Discussion by lckr** [brandonp2412/Flexify#108] [Q&A]
Hi, is it currently possible to manually specify rest time between different sets of exercises ? 
My use case is that I would want to add rest times of different length to my workout plan, e.g. longer rest times between sets of heavy compound movements and shorter rest times between sets of accessory exercises.

**Comment by brandonp2412** [comment]
Yes. Graphs -> Tap exercise -> Tap edit -> Set rest minutes/seconds
